extensions/amp-form/0.1/test/test-form-dirtiness.js

```diff
@@ -36,6 +37,12 @@ describes.realWin('form-dirtiness', {}, env => {
   beforeEach(() => {
     doc = env.win.document;
     form = getForm(doc);
+
+    sandbox.stub(Services, 'platformFor').returns({
```

Comment:
Why do we need this?

Reply:
For `createFormDataWrapper` :

https://github.com/ampproject/amphtml/blob/14a640bb7eda7098f0a4026a830434c331609a06/src/form-data-wrapper.js#L34

Reply:
FWIW this is how `createFormDataWrapper` is stubbed in the codebase.

https://github.com/ampproject/amphtml/blob/c4a663d0ba13d0488c6fe73c55dc8c971ac6ec0d/test/unit/test-xhr-fetch-polyfill.js#L109-L113